src/test/kotlin/com/igorwojda/string/longestword/challenge.kt

```diff
@@ -4,7 +4,18 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun longestWord(str: String): String {
-    TODO("not implemented")
+
+    val result = str.replace(("[^\\w]").toRegex(), " ")
+    val rawArray=result.split(" ")
```

Comment:
Transform this into a chain of collection operators:
```
result.split(" ")
  .toList()
  .filter...
  .maxBy...
```

Reply:
return result.split(" ")
        .toList()
        .filter { it.isNotBlank() }
        .maxBy{it.length}?:""